### PR TITLE
Update Release Notes format

### PR DESCRIPTION
**Pull request by sivakku**
- Add release date to the update version line.
- Fix formatting of Bug fixes to explain and then link the issue subsequently.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-### Fixes 
-* [Issue #362](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/362) Fixes an issue where environmental path delimiter can invalidate the path on Windows.
-* Fixes an issue where Cloud SDK sometimes gets corrupted on Windows since there is reentry while installing dependencies.
+### Bug Fixes 
+* Fix an issue where dependencies path gets invalidated on Windows. [#362](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/362) 
+* Fix an issue where Cloud SDK in the path gets corrupted on Windows since there is reentry while installing dependencies. [#347](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/347)
 
-## Version 1.7.1 Release Notes
+## Version 1.7.1 Release Notes: December 09, 2020
```
Comment (kschaab) [inline CHANGELOG.md:11]: Wasn't this multi-lined in the proposal (header followed by the date)?

**Review by kschaab — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (sivakku) [reply CHANGELOG.md:11]: That doc format is WIP and not yet finalized, currently following the cpp extension and that seemed more readable.

Comment (sivakku) [reply CHANGELOG.md]: Chairy updated the recommendation in the doc to "version (date)" format, the update is to reflect to that format.

**Review by kschaab — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -186,28 +185,28 @@ Updated the extension to be compatible with the latest version of Visual Studio
...
 
-* [Issue #176:](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/176) Cloud Code will no longer continually switch user context to the output window!
-* [Issue #201:](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/201) Fixed an issue where the Logs Viewer hangs when you don’t have permission to list all namespaces.
+* Cloud Code will no longer continually switch user context to the output window. [#176](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/176) 
+* Fixed an issue where the Logs Viewer hangs when you don’t have permission to list all namespaces. [#201](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/201) 
 * Fixed an issue where Open with Cloud Code would not clone a repository in some cases.
 * Fixed an issue where Cloud Shell would fail to create an SSH configuration when the gcloud command-line tool prompted for survey participation.
 
-## Version 1.1.2
+## Version 1.1.2: Feb 27, 2020
```
Comment (kschaab) [inline CHANGELOG.md:195]:
```suggestion
## Version 1.1.2 (Feb 27, 2020)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -151,19 +150,19 @@ We are pleased to announce [Cloud Run](https://cloud.google.com/run) support in
...
-* [#203](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/203) Add a guidance message in case no Kubernetes manifest are present
+* Add support for Google Cloud Run. [#231](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/231)
+* Hanging while installing dependencies. [#227](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/227) 
+* Cannot start debugger for .NetCore project. [#225](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/225) 
+* GKE Explorer does not work in windows 10. [#224](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/224)
+* Add Configuration to Existing Project is broken. [#223](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/223)
+* Add a guidance message in case no Kubernetes manifest are present. [#203](https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/203) 
 
-## Version 1.2.1
+## Version 1.2.1: March 25, 2020
```
Comment (kschaab) [inline CHANGELOG.md:160]:
```suggestion
## Version 1.2.1 (March 25, 2020)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -151,19 +150,19 @@ We are pleased to announce [Cloud Run](https://cloud.google.com/run) support in
...
 
 Updated the extension to be compatible with the latest version of Visual Studio code. Fixed an issue introduced by the latest update that caused specific terminal commands to hang.
 
 
-## Version 1.2.0
+## Version 1.2.0: March 02, 2020
```
Comment (kschaab) [inline CHANGELOG.md:165]:
```suggestion
## Version 1.2.0 (March 02, 2020)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -186,28 +185,28 @@ Updated the extension to be compatible with the latest version of Visual Studio
...
+## Version 1.1.2: Feb 27, 2020
 
 ### Notable Fixes
 
 * Fixed an issue in the "Open in Cloud Code" feature
 * Fixed an issue where logs in "Cloud Code Yaml Support" output channel will grab focus
 * Minor bug fixes
 
-## Version 1.1.1
+## Version 1.1.1: Feb 27, 2020
```
Comment (kschaab) [inline CHANGELOG.md:203]:
```suggestion
## Version 1.1.1 (Feb 27, 2020)
```

**Review by kschaab — CHANGES_REQUESTED** [review]

**Review by kschaab — COMMENTED** [review]
1.1.0 is January 30, 2020
1.0.1 is November 21, 2019
0.0.13 is November 7, 2019

File: CHANGELOG.md (hunk 2 above)
Comment (sivakku) [reply CHANGELOG.md:195]: Good catch! missing string replaces, updated the rest of them as well.

**Comment by sivakku** [comment]
> 1.1.0 is January 30, 2020
> 1.0.0 is November 21, 2019
> 0.0.13 is November 7, 2019

Nice! couldn't figure when those were released. Added to the change.

**Review by kschaab — APPROVED** [review]

**Pull request merged**